oshi-core/src/main/java/oshi/hardware/User.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Oshi (https://github.com/oshi/oshi)
+ *
+ * Copyright (c) 2010 - 2018 The Oshi Project Team
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Maintainers:
+ * dblock[at]dblock[dot]org
+ * widdis[at]gmail[dot]com
+ * enrico.bianchi[at]gmail[dot]com
+ *
+ * Contributors:
+ * https://github.com/oshi/oshi/graphs/contributors
+ */
+package oshi.hardware;
+
+/**
+ * @author: BilalAM
+ */
+public interface User {
```

Comment:
I'm wondering if this should be focused on the currently logged in user or if we can make this more general to apply to any computer user; there could be an "isCurrentUser" boolean.  But this somewhat reminds me of the OSHI process as it's running. It may have its own PID but that's just a small picture of the rest of the system.   The current user is of interest (as is the current process) but it's also useful to know other users with accounts on the computer.